raiden-ts/src/messages/utils.ts

```diff
@@ -2,15 +2,22 @@
 import { ThrowReporter } from 'io-ts/lib/ThrowReporter';
 import { isLeft } from 'fp-ts/lib/Either';
 
-import { Signer } from 'ethers';
+import { ethers, Signer } from 'ethers';
```

Comment:
You could import `RLP` directly from `ethers/utils`. `Importing the whole `ethers` module again only for `ethers.utils.RLP.encode` isn't good.

Reply:
I completely missed that thank you, I will fix it :)